### DISCUSSION TITLE
Customize Dependency Management Columns

### DISCUSSION THREAD

**Discussion by capeflynn** [bsorrentino/maven-confluence-plugin#298] [Q&A] [answered]
How do you customize the columns in the dependencies report? Is it possible to enable the licensing or other columns in the dependencies report from the command line? Thank you for your help.

**Comment by bsorrentino** [comment, marked as answer]
Hi @capeflynn 

Unfortunately the dependencies table is hard coded and it is possible to configure only by code.

Probably could be interesting create an independent module for that, so the customisation should be easier
